Add comments about typography principles (#453)
pxblue/doc-it: src/docs/style/typography.mdx

Unlike Material Design, the primary font for Power Xpert Blue is [Open Sans](https://fonts.google.com/specimen/Open+Sans?selection.family=Open+Sans) (licensed under [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)). This version contains the complete 897 character set, which includes the standard ISO Latin 1, Latin CE, Greek and Cyrillic character sets. Open Sans was optimized for print, web, and mobile interfaces, and has excellent legibility characteristics in its letter forms.

<TOC
    anchors={[
        { title: 'Introduction', hash: '#typography-guidelines' },
        { title: 'Type Scale', hash: '#type-scale' },
        { title: 'Principles', hash: '#principles' },
        { title: 'Customization', hash: '#customization' },
        { title: 'Internationalization', hash: '#internationalization' },
    ]}
/>
<Divider />

# Type Scale

PX Blue provides a type scale that includes a range of styles to support the needs of your application. All styles and fonts are available in the design library. As PX Blue continues to grow, additional font patterns will be added.

<TypographyTable />

# Principles

> Font variations can create a hierarchy in your information display that guides your user's attention. This involves changing the font's size, weight, and / or color.

Reviewer: ```suggestion
You should use font variations to establish an information hierarchy in your display that guides users' attention from the most important details to the least important. Creating this information hierarchy involves using multiple font properties, including size, weight, and / or color.
```

<ImageGrid
    images={[TextHierarchy1, TextHierarchy2, TextHierarchy3, TextHierarchy4]}
    caption={[

>         'You can manipulate many different aspects of your typography to indicate hierarchy.',

Reviewer: ```suggestion
        'You can manipulate different typography properties to establish hierarchy.',
```

        '1: Font size',
        '2: Font weight',
        '3: Font color',

>         '4: Combined',

Reviewer: Super picky, but the first three are nouns and the fourth is an adjective
```suggestion
        '4: Combination',
```

    ]}
/>

> Before starting on your design, you should understand what contents are important from your user's perspective.

Reviewer: ```suggestion
As you are crafting your designs, you need to understand which content is the most important from your users' perspective and make that the easiest to find on the page.
```

<ImageGrid
    images={[CompeteAttention1, CompeteAttention2]}
    caption={[

>         '1: Do not rely solely on font weight changes to indicate an active state. Do not let auxiliary text (the circled numbers on the right) distract your user.',

Reviewer: ```suggestion
        '1: Do not rely solely on font weight changes to indicate an active state. Avoid having auxiliary information (e.g., the circled numbers on the right) compete for users' attention.',
```

        '2: Use clues like background color and radio buttons to clearly communicate the state.',
    ]}
/>

> Be careful with font colors too, as they can raise [accessibility issues](/design/accessibility).

Reviewer: ```suggestion
When changing font colors, make sure to take into consideration the [accessibility](/design/accessibility) of your chosen colors.
```

<ImageGrid
    images={[StatusColor1, StatusColor2]}
    caption={[

>         '1: Be careful when using status color for a large body of text, as they have decreased legibility compared to those with neutral colors.',

Reviewer: ```suggestion
        '1: Be careful when using status colors for a large body of text, as they have reduced legibility compared to standard text colors.',
```
